web/src/stream_edit.js

```diff
@@ -665,21 +666,36 @@ export function initialize() {
         return true;
     });
 
+    const stream_save_changes = ($save_button) => {
```

Comment:
If this is just for stream privacy changes, let's call this `save_stream_privacy_settings`.

If this applies to other settings too ... then probably we don't want the dialog widget for those, as that'd be rather misleading.

Reply:
Thanks, that makes sense since it can be a bit confusing!